utils/parts.stanza

```diff
@@ -65,20 +65,15 @@ with :
 
 ; Parse parts conditionally, based on their category.
 public defn parse-component (json: JObject) -> Component :
-  val component-code = json["component"] as JObject
-  match(component-code["landpattern"]):
-    (lp:JObject):
-      ; If the landpattern is included, this part should have everything it needs.
+  match(json["category"]):
+    (c:String):
+      switch(c):
+        "capacitor": Capacitor(json)
+        "inductor": Inductor(json)
+        "resistor": Resistor(json)
+        else: Part(json)
+    (c:JNull):
```

Comment:
Is there a case it can be null?

Reply:
Yes, if it's a part we can't identify in our "top 10", we leave it empty.

In theory, we could expand the category list so everything has a value - but for now, we only handle the common cases.